src/utils/emailTemplates.ts

```diff
@@ -237,12 +239,25 @@ export const htmlUserCreationRequestWithDate = (data: {
                             : ''
                     }
                     ${
-                        userCreationRequest.coordinator
-                            ? `
+                        userCreationRequest.coordinators && userCreationRequest.coordinators.length
```

Comment:
why not use `nullable` helper function?

Reply:
Will replace ternarys by it in https://github.com/taskany-inc/crew/issues/1097